apps/settings/test/marionette/keyboardtestapp/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Keyboard APP Test</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="textInput"/>
```

Comment:
Any reason why we can't reuse the test app in `apps/keyboard/test`?

(I am fine if you have talk to Ray and decided this is easier...)